src/murfey/server/api/__init__.py

```diff
@@ -1060,22 +1064,18 @@ async def request_tomography_preprocessing(
     core = Path(*Path(proc_file.path).parts[: visit_idx + 1])
     ppath = Path("/".join(secure_filename(p) for p in Path(proc_file.path).parts))
     sub_dataset = "/".join(ppath.relative_to(core).parts[:-1])
+    extra_path = machine_config.processed_extra_directory
     mrc_out = (
         core
         / machine_config.processed_directory_name
         / sub_dataset
+        / extra_path
         / "MotionCorr"
+        / "job002"
+        / "Movies"
         / str(ppath.stem + "_motion_corrected.mrc")
     )
     mrc_out = Path("/".join(secure_filename(p) for p in mrc_out.parts))
-    ctf_out = (
-        core
-        / machine_config.processed_directory_name
-        / sub_dataset
-        / "CTF"
-        / str(ppath.stem + "_ctf.mrc")
-    )
-    ctf_out = Path("/".join(secure_filename(p) for p in ctf_out.parts))
```

Comment:
What did this used to do?

Reply:
This used to create the name of the ctf output file. In SPA the MotionCorr service works that file name out itself, so tomography can move to the same method.